src/state/shell/onboarding.tsx

```diff
@@ -82,12 +82,16 @@ export function Provider({children}: React.PropsWithChildren<{}>) {
 
   React.useEffect(() => {
     return persisted.onUpdate(() => {
-      dispatch({
-        type: 'set',
-        step: persisted.get('onboarding').step as OnboardingStep,
-      })
+      const next = persisted.get('onboarding').step
+      // TODO we've introduced a footgun
+      if (state.step !== next) {
```

Comment:
Hehe this was cyclically updating in the background. Wonder if we can protect against this better at a higher level?

Reply:
mm that was on me, I think the only way to solve it is to avoid fires in response to this event -- the `type: set` needs a `silent: true` param